### ISSUE TITLE
[usability] error, but marked green

### ISSUE THREAD

**Issue by kwladyka** [argoproj/argo-cd#1587]
**Describe the bug**
![image](https://user-images.githubusercontent.com/3903726/57395716-1cf9aa00-71c9-11e9-857e-a333ade266c3.png)

![image](https://user-images.githubusercontent.com/3903726/57395731-26831200-71c9-11e9-819c-24b84f6a8f58.png)

it is Uknown endless

error:
```
rpc error: code = Unknown desc = Error: argo/argo-events chart not found in repo https://argoproj.github.io/argo-helm
```

**To Reproduce**
helm chart with
requirements.yaml
```
dependencies:
  # https://github.com/argoproj/argo-helm/blob/master/charts/argo-events/Chart.yaml
  - name: argo/argo-events
    repository: https://argoproj.github.io/argo-helm
    version: 0.4.1
```

values.yaml empty

**Expected behavior**
Marked on red as failed.

**Comment by alexec** [comment]
I'm not sure if you need to add this chart to the `ConfigMap`:

https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#helm-chart-repositories

Might be something to try? Kind of tied up at the moment to properly investigate.

**Comment by kwladyka** [comment]
Yes I needed to do it, but I made a mistake:
```
dependencies:
  # https://github.com/argoproj/argo-helm/blob/master/charts/argo-events/Chart.yaml
  - name: argo/argo-events
    repository: https://argoproj.github.io/argo-helm
    version: 0.4.1
```

But should be:
```
dependencies:
  # https://github.com/argoproj/argo-helm/blob/master/charts/argo-events/Chart.yaml
  - name: argo-events # <---------
    repository: https://argoproj.github.io/argo-helm
    version: 0.4.1
```

But it is not about why this error appear. It is about how this error appear in UI. I saw it only because I was working on it and watching it. Normally I would not see it from main view, because this not error. Endless Unknown and green status.

**Comment by alexec** [comment]
Fixed in v1.2
